runtime/integration-tests/src/xcm/polkadot/tests/asset_registry.rs

```diff
@@ -108,7 +108,7 @@ fn register_foreign_asset_works() {
 fn register_tranche_asset_blocked() {
 	Centrifuge::execute_with(|| {
 		let meta: AssetMetadata<Balance, CustomMetadata> = AssetMetadata {
-			decimals: 12,
+			decimals: 18,
```

Comment:
?

Reply:
My bad, forgot to revert. I was under the impression due to a discussion on the solidity side that tranche tokens were always 18 decimals so I thought this was wrong. Will revert 👍